mk/llvm.mk

```diff
@@ -95,6 +95,8 @@ $(foreach host,$(CFG_HOST), \
 # This can't be done in target.mk because it's included before this file.
 define LLVM_LINKAGE_DEPS
 $$(TLIB$(1)_T_$(2)_H_$(3))/stamp.rustc_llvm: $$(LLVM_LINKAGE_PATH_$(2))
+RUSTFLAGS$(1)_rustc_llvm_T_$(3) += $$(shell $$(LLVM_CONFIG_$(3)) --components | tr '-' '_' |\
```

Comment:
> Can this use $(LLVM_COMPONENTS) instead of re-invoking llvm-config here?

I don't see why not.

> This also seems to disagree currently in terms of which llvm-config is run?

What do you mean?